package/Dockerfile

```
@@ -363,11 +363,12 @@ ARG COMMIT
 ARG TAGS="k8s"
 ARG LINKFLAGS="-extldflags -static -s"
 ARG LDFLAGS="-X github.com/rancher/rancher/pkg/version.Version=${VERSION} -X github.com/rancher/rancher/pkg/version.GitCommit=${COMMIT} ${LINKFLAGS}"
+ARG EXTRAFLAGS=""
```

The default is already empty, no need for that ="".

```suggestion
ARG GCFLAGS
```